### ISSUE TITLE
[Feature] updateAxisPointer事件没有找到对应的api

### ISSUE THREAD

**Issue by dmcisgood** [apache/echarts#18936]
### What problem does this feature solve?

在案例中看到了一个updateAxisPointer事件，但是在api文档中没有找到相关的信息

### What does the proposed API look like?

想知道这个信息在哪里

**Comment by helgasoft** [comment]
Yes, API docs do not mention _updateAxisPointer_, but a repo search will find calls like:
https://github.com/apache/echarts/blob/586ca04b5c925d455db35324b2d4ecd746b93a27/src/component/tooltip/TooltipView.ts#L436
This format works:
```chart.dispatchAction({ type: 'updateAxisPointer', seriesIndex: 0, dataIndex: 44 });```

**Comment by badspider7** [comment]
只能传dataIndex吗，如果我x轴的type为time是一个时间戳，如何让他跟着传进来的时间戳走呢，有大佬有类似的例子吗

**Comment by dilipkumar6306316-art** [comment]
This issue causes a complete rendering failure of child nodes upon hover. I've found that using shallowRef for the chart instance and setting select: { disabled: true } for parent components (Blob/Pool) mitigates the issue where children nodes disappear.
